oci/oci.go

```diff
@@ -85,6 +91,7 @@ type Runtime struct {
 	trustedPath              string
 	untrustedPath            string
 	trustLevel               string
+	runtimes                 map[string]RuntimeHandler
```

Comment:
all this confusion boils down to the fact that a struct named Runtime holds a Runtimes map, omg
I'm just saying we need to fix this :) not in this PR maybe

Reply:
I'll take care of this in a follow up